pom.xml

```diff
@@ -792,6 +793,12 @@
                 <artifactId>resilience4j-ratelimiter</artifactId>
                 <version>${resilience4j.version}</version>
             </dependency>
+            <!-- https://mvnrepository.com/artifact/org.bouncycastle/bcpkix-jdk15on -->
+            <dependency>
+                <groupId>org.bouncycastle</groupId>
+                <artifactId>bcpkix-jdk15on</artifactId>
```

Comment:
Actually this versions is intended for Java 5 and version 1.70 is from 2021. From their site https://www.bouncycastle.org/latest_releases.html we can use artifactId: jdk18on and version 1.71.1 https://mvnrepository.com/artifact/org.bouncycastle/bcpkix-jdk18on